nixos/options.tt

```diff
@@ -1,4 +1,4 @@
-[% WRAPPER layout.tt title='Search NixOS options' menu='nixos' %]
+[% WRAPPER layout.tt title='Search NixOS options' menu='nixos' hideSearch=1 %]
```

Comment:
i think it makes sense to leave the search there for consistency

Reply:
I agree, we could add to search on google by default (`https://www.google.com/search?q=site%3Anixos.org+nix&oq=site%3Anixos.org+<QUERY>`). and then also have an option to search for packages and option in the dropdown as it is now. 

I hope to change this soon once we remove current implementation of packages/options search.

Reply:
I would consider leading visitors to Google controversial. Then we could directly use Google Analytics.

I personally use startpage.com for web search and it's quiet popular. It is just a proxy for Google and you should get the same results without personalization and location based ones. Search URL is `https://www.startpage.com/do/dsearch?query=site%3Anixos.org+nix`.

![Screenshot from 2020-04-22 20-09-11](https://user-images.githubusercontent.com/91113/80018394-3c6f0d00-84d6-11ea-94d4-3bd7ea12ba71.png)

But i don't know if we can agree on that.

Another option is local search using JS like [Tipue](https://tipue.com/search/) or [lunrjs](https://lunrjs.com/). I use the first on my website which has 403 pages (number of pages, not HTTP code :D). It works great. The NixOS website has probably less pages after the redesign, or should it also include the manuals?

I think the search for packages/options should have a different place. A separate page was fine. I always type "n" in my browser and select the page for the packages or options search.

Reply:
For now I will leave it as it is. Currently we don't have a site search and we can add it in follow up PR is we decide to. But that means we will also have to select the search engine, etc.. which is another can of worms.